source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -1,3 +1,61 @@
-===========================
-Header files and namespaces
-===========================
+============
+Header files
+============
+
+oneDAL public identifiers are represented in the following header files:
+
+.. list-table::
+   :widths:  10 50
+   :header-rows: 1
+
+   * - header file
+     - description
+   * - ``onedal/onedal.hpp```
+     - Main header file of oneDAL library.
+   * - ``onedal/algo/%ALGORITHM%/%ALGORITM%.hpp``
+     - An algorithm header file for a particular algorithm.
```

Comment:
I'd suggest adding a blank line between lines 16 and 17 to have two paragraphs here